charts/invenio/templates/terminal-deployment.yaml

```diff
@@ -0,0 +1,169 @@
+          {{- include "invenio.config.queue" . | nindent 10 }}
+          {{- include "invenio.config.database" . | nindent 10 }}
+          {{- with .Values.invenio.extraEnvVars }}
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
+          {{- with .Values.terminal.extraEnvVars }}
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
+          {{- include "invenio.config.sentry" . | nindent 10 }}
+          {{- include "invenio.config.datacite" . | nindent 10 }}
+          {{- if .Values.invenio.remote_apps.enabled }}
+          {{- range .Values.invenio.remote_apps.credentials }}
+          - name: {{ default (printf "INVENIO_%s_APP_CREDENTIALS" .name) }}
+            valueFrom:
+              secretKeyRef:
+                name: {{ $.Values.invenio.remote_apps.secret_name }}
+                key: {{ default (printf "INVENIO_%s_APP_CREDENTIALS" .name) }}
+          {{- end }}
+          {{- end }}
+          {{- range .Values.invenio.extra_env_from_secret }}
+          - name: {{ .name }}
+            valueFrom:
+              secretKeyRef:
+                name: {{ .valueFrom.secretKeyRef.name }}
+                key: {{ .valueFrom.secretKeyRef.key }}
+          {{- end }}
+          livenessProbe: {{- tpl (.Values.terminal.livenessProbe | toYaml) $ | nindent 12 }}
+          readinessProbe: {{- tpl (.Values.terminal.readinessProbe | toYaml) $ | nindent 12 }}
+          startupProbe: {{- tpl (.Values.terminal.startupProbe | toYaml) $ | nindent 12 }}
+          volumeMounts:
+            {{- if .Values.persistence.enabled }}
+            - mountPath: /opt/invenio/var/instance/data
+              name: shared-volume
+              subPath: data
+            {{- end }}
+            {{- if .Values.kerberos.enabled }}
+            - name: kerberos-credentials-cache
+              mountPath: /tmp
+            {{- end }}
+            {{- range $key, $value := .Values.invenio.vocabularies }}
+            - name: vocabularies
+              mountPath: "/opt/invenio/var/instance/app_data/vocabularies/{{ $key }}"
+              subPath: "{{ $key }}"
+            {{- end }}
+        {{- if .Values.kerberos.enabled }}
+        - name: kerberos-credentials
+          image: {{ required "Missing .Values.kerberos.image" .Values.kerberos.image }}
+          args: {{ .Values.kerberos.args }}
+          envFrom:
+            - secretRef:
+                name: {{ .Values.kerberos.secret_name }}
+          resources: {{- toYaml .Values.kerberos.resources | nindent 12 }}
+          {{- if .Values.kerberos.readinessProbe }}
+          readinessProbe: {{- toYaml .Values.kerberos.readinessProbe | nindent 12 }}
+          {{- end }}
+          {{- if .Values.kerberos.livenessProbe }}
+          livenessProbe: {{- toYaml .Values.kerberos.livenessProbe | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: kerberos-credentials-cache
+              mountPath: /tmp
+        {{- end }}
```

Comment:
I'm unfamiliar with kerberos. Why is it needed here?

Reply:
BWT, I simply copied everything we have from worker/web.